miso-web/src/main/webapp/scripts/bulk_sample.js

```diff
@@ -1337,4 +1339,25 @@ BulkTarget.sample = (function($) {
     }
   }
 
+  function separateExternalNames(names) {
+    if (!names) {
+      return [];
+    }
+    return names.split(',').map(function(name) {
+      return name.trim();
+    }).filter(function(name) {
+      return !!name;
+    });
+  }
+  function anyMatch(arr1, arr2) {
+    for (var i1 = 0; i1 < arr1.length; i1++) {
```

Comment:
What do you think about making this implementation? (I find it more semantic/readable, and want to know what you think)
```
let matches = arr1.filter(a => arr2.includes(a));
return matches.length > 0
```

Reply:
Yes, that is better, thanks! I improved it further by returning `arr1.some(...)` instead of filtering and checking length